pkgs/tools/system/auto-epp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, python3 }:
```

Comment:
Would you be willing to move your package to the new `by-name` set by the way? AFAIK it's not obligatory just yet, but many new packages are shifting towards using this.

Reply:
sure, any docs or example i can read ?

Reply:
Sure! There's https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
if i understand properly, i just need to move my default.nix file  to  pkgs/by-name/au/auto-epp/package.nix  and remove it from the top-level/all-packages.nix ?

Reply:
Yes, I think so

Reply:
ok, done